### PR TITLE
 NOD and SC | Update Max and Min filesize error content

### PR DESCRIPTION
**Pull request by rocio-desantiago**
## Summary

- _(Summarize the changes that have been made to the platform)_
   > content updates approved by CAIA in [comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/74457#issuecomment-1936004246) 
  > Delete button doesn't show up in staging/production therefore the language does not apply for the buttons that do show up in staging/prod which are `Add another document` and `Cancel`
- _(If bug, how to reproduce)_
   > Not a bug
- _(What is the solution, why is this the solution)_
   > Content updates per CAIA
- _(Which team do you work for, does your team own the maintenance of this component?)_
   > Benefits Team 1 - Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_
   > n/a

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#76161
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/va.gov-team#74457

## Testing done

- _Describe what the old behavior was prior to the change_
    > No behavior change
- _Describe the steps required to verify your changes are working as expected_
    > -  Upload a large file that exceeds 100MB
           - You should see the following error: `We couldn’t upload your file because it’s too large. File size must to be less than 100 MB`
    > -  Upload a small file less than 1KB 
            - You should see the following error: `We couldn’t upload your file because it’s too small. Please delete this file. Try uploading a file that’s 1KB or more.`
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Error Message | Large-small-Before | large-tiny-after-updated |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by Andrew565** [comment]
PR has failed a required step, opting not to review at this time. - Tier 2/FE CoP

**Comment by rocio-desantiago** [comment]
@Andrew565 
> PR has failed a required step, opting not to review at this time. - Tier 2/FE CoP

Fixed the unit test related to the update.

**Comment by Andrew565** [comment]
PR has failing required step (QA Merge Status), opting not to review at this time. - Tier 2/FE CoP

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 1]
```diff
@@ -89,6 +89,7 @@ export default Object.freeze({
   facilityLocatorShowOperationalHoursSpecialInstructions:
     'facility_locator_show_operational_hours_special_instructions',
   fileUploadShortWorkflowEnabled: 'file_upload_short_workflow_enabled',
+  findFormsShowPdfModal: 'find_forms_show_pdf_modal',
```
Comment (Mottie) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:92]: Not sure where this change came from?

**Comment by rocio-desantiago** [comment]
@Andrew565 thank you for that note. It is in good shape now.

**Review by Mottie — APPROVED** [review]

**Review by ataker — APPROVED** [review]

**Pull request merged**